### PR TITLE
Add test config minimum_interval

### PR DESCRIPTION
**Pull request by pierreprinetti**
Bumps test-infra (one [code change](https://github.com/openshift/ci-tools/pull/2980/commits/6c0b19cba27fcd2e1b586b11251ecd766d2812b4#diff-c48fabca0aa1696f8e0cd90a07e688bff1a5be2e22eb2f44fef5098ff271f0daR60-R75) was necessary).

Based on: https://github.com/kubernetes/test-infra/pull/26743

Implements [OSASINFRA-2982](https://issues.redhat.com//browse/OSASINFRA-2982)

### CODE REVIEW

File: cmd/release-job-migrator/main.go [hunk 1]
```diff
@@ -522,6 +522,10 @@ func run(o options) error {
 				intervalCopy := periodic.Interval
 				interval = &intervalCopy
 			}
+			if periodic.MinimumInterval != "" {
+				minimumIntervalCopy := periodic.MinimumInterval
+				interval = &minimumIntervalCopy
```
Comment (smg247) [inline cmd/release-job-migrator/main.go:527]: you want to set `minimumInterval` here instead

**Comment by pierreprinetti** [comment]
This will have to wait for https://github.com/kubernetes/test-infra/pull/27304
/hold

**Comment by pierreprinetti** [comment]
`UPDATE=true make test` still geenrates `minimun_interval` with a typo. Perhaps next week?

**Comment by pierreprinetti** [comment]
/hold cancel
Now the problem seems to be that 

>  The ci-chat-bot repo vendors test-infra at e655beaa3871, which is older than the ci-tools vendor at 0e2e39caed68

Looks like I'll have to wait for a bump in ci-chat-bot?

**Comment by smg247** [comment]
> > The ci-chat-bot repo vendors test-infra at [e655bea](https://github.com/openshift/ci-tools/commit/e655beaa387138967f72065b33e6dc264accf712), which is older than the ci-tools vendor at 0e2e39caed68
> 
> Looks like I'll have to wait for a bump in ci-chat-bot?

Yes, `ci-chat-bot` and `release-controller` cannot have an older version of `test-infra` than `ci-tools`. You will have to bump both of them to the same version.

**Review by smg247 — CHANGES_REQUESTED** [review]
You'll also need to add a validation like we have for: https://github.com/openshift/ci-tools/blob/7837783c02f4e86594f6048f3c28614816f96472/pkg/validation/test.go#L163-L165 to verify that this field isn't set with `cron` or `interval`

File: pkg/steps/release/promote.go [hunk 2]
```diff
@@ -60,7 +60,7 @@ func (s *promotionStep) run(ctx context.Context) error {
 	opts := []PromotedTagsOption{
 		WithRequiredImages(s.requiredImages),
 	}
-	if refs := s.jobSpec.MainRefs(); refs != nil {
+	if refs := s.jobSpec.Refs; refs != nil {
```
[the remark this answers is not in the document]
Comment (pierreprinetti) [reply pkg/steps/release/promote.go:63]: `MainRefs()` [is not an exported method any more](https://github.com/kubernetes/test-infra/commit/51b20129359466d7c7ce3bb8457b7a648b29ecfa#diff-3062bdb94a540e9fb07c50039ad8b2190ecb3c2e38cf190b03dfc81afcff781fR196).

**Comment by pierreprinetti** [comment]
@smg247 Validation added, thanks! PTAL

File: pkg/steps/release/promote.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (pierreprinetti) [reply pkg/steps/release/promote.go]: Done!

**Pull request merged**